_notices/rsn0025.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 25 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Support for Ubuntu 22.04 in v23.02"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.12"
+notice_created: 2021-11-21
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-11-21
+---
+
+## Overview
+
+With the depracation of `Ubuntu 18.04` announced in [RSN 24](/notices/rsn0024), development
+effort has been redirected to support `Unbuntu 22.04` in `v23.02` nightly releases.
+
+## Status
+
+Dropping support for Ubuntu 18.04 is an ongoing effort.
+
+## Impact
+
+Users should consider switching to any of the following supported operating systems for any `rapidsai/rapidsai` image:
```

Comment:
I don't think it makes sense to tell users to switch to a supported platform in a message about bringing a new platform online. (Also, the same note as above on Rocky Linux 8.)